demos/keyboard-nav/index.html

```diff
@@ -214,7 +215,13 @@ <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
       } else if (newNode) {
         return treeTraversal(newNode);
       } else {
-        return findSiblingOrParent(node);
+        var siblingOrParent = findSiblingOrParent(node);
```

Comment:
Hmm...Just a note, to get to this line I think node.in() and node.next() had to return null, so you could pass in node.out() to save an iteration.